File: 240417/알파벳과 숫자 출력/alphabet-and-numeric-output.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    char alphabet[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    char number[] = "123456789";

    int alphabetIndex = 0;
    int numberIndex = 0;

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n - i; j++) {
            printf("%c ", alphabet[alphabetIndex]);
            alphabetIndex = (alphabetIndex + 1) % 26;
        }
        for (int j = 0; j < i + 1; j++) {
            printf("%c ", number[numberIndex]);
            numberIndex = (numberIndex + 1) % 9;
        }
        printf("\n");
    }

    return 0;
}

File: 240417/알파벳과 숫자 출력/alphabet-and-numeric-output.kt
fun main() {
    val n = readLine()!!.toInt()

    val alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    val number = "123456789"

    var alphabetIndex = 0;
    var numberIndex = 0;

    for(i in 0 until n) {
        for(j in 0 until n-i) {
            print("${alphabet[alphabetIndex]} ")
            alphabetIndex = (alphabetIndex + 1) % 26
        }
        for(j in 0 until i+1) {
            print("${number[numberIndex]} ")
            numberIndex = (numberIndex + 1) % 9
        }
        println()
    }
}